attrd/main.c

```diff
@@ -121,15 +121,16 @@ attrd_cib_destroy_cb(gpointer user_data)
     return;
 }
 
-static cib_t *
+static int
```

Comment:
Really don't agree with anything in this patch :)

Reply:
Yeah ok that commit was purely style :)

Reply:
Or where you referring to the PR as a whole? It makes sense to me that attrd would maintain control over the transient attributes, and at cluster start-up, it avoids race conditions.